ocs_ci/utility/utils.py

```diff
@@ -5041,3 +5040,32 @@ def sum_of_two_storage_sizes(storage_size1, storage_size2, convert_size=1024):
     size = size1 + size2
     new_storage_size = f"{size}{unit}"
     return new_storage_size
+
+
+class CustomJSONEncoder(json.JSONEncoder):
+    """
+    Custom JSON encoder to handle set objects
```

Comment:
Please describe in docstring what this custom json encoder does differently and maybe change its name to reflect it. Also please consider if it should be in its own file (e.g. ocs_ci/utility/json.py).

Reply:
done